### ISSUE TITLE
newLineKind = system becomes LF on Windows 11

### ISSUE THREAD

**Issue by Auxx** [dprint/dprint-plugin-typescript#499]
When I set `"newLineKind": "system",` it turns to forces all files to be formatted with `LF` instead of `CRLF` on Windows 11.

dprint-plugin-typescript version: 0.83.0

NodeJS version: 18.13.0

The source code is irrelevant as it happens for all TS/JS files.

**Comment by dsherret** [comment]
This can be fixed now by upgrading to dprint 0.43.1 and setting this config at the top level. It will be removed from this plugin's config in the future and at that time I'll close this issue.

**Comment by jsejcksn** [comment]
@dsherret Will this change solve the same behavior in Fresh projects? (Ref: This Stack Overflow question: [deno-fmt set line ending behaviour lf/crlf](https://stackoverflow.com/q/77584474))

**Comment by dsherret** [comment]
No, nothing to do with deno fmt. See https://github.com/denoland/deno/issues/20194
